src/tests/tests/utils.js

```diff
@@ -65,6 +65,17 @@
     assert.ok(!badger.isPrivacyBadgerEnabled("example.com"), "disabled for site");
     badger.enablePrivacyBadgerForOrigin("example.com");
     assert.ok(badger.isPrivacyBadgerEnabled("example.com"), "enabled for site");
+
+    // whitelist wildcard cases
+    badger.disablePrivacyBadgerForOrigin('*.mail.example.com');
+    assert.ok(badger.isPrivacyBadgerEnabled('www.example.com'), "enabled for site");
+    assert.ok(badger.isPrivacyBadgerEnabled('web.stuff.example.com'), "enabled for site");
+    assert.ok(!badger.isPrivacyBadgerEnabled('web.mail.example.com'), "disabled for site");
```

Comment:
We could use a more specific [QUnit assertion](https://api.qunitjs.com/assert/) here (and below): `assert.notOk`.